app/javascript/components/public_health/Import.js

```diff
@@ -79,6 +81,18 @@ class Import extends React.Component {
         });
       })
       .catch(err => {
+        let validation_errors = [];
+        Object.keys(err.response.data).forEach(prop => {
+          validation_errors.push({
+            prop,
+            message: err.response.data[String(prop)],
+          });
+        });
+        const patientIndex = this.state.patients.findIndex(statePatient => _.isEqual(statePatient, patient.patient));
+        let patients = this.state.patients;
+        patient.patient.validation_errors = validation_errors;
```

Comment:
I think we can make the code less confusing here (`patient.patient`) by changing some lines at the start of this method. Specifically, setting `patient` to the actual Patient data rather than `{ patient: { ...data } }` and then updating the object sent in the POST to have the right structure.

So line 63 could be: 
`const patientData = { ...data };`

and line 68 could be:
`data: { {patient: ...patientData}, bypass_duplicate: bypass },`

and variables could be updated accordingly.

Reply:
> Have we shown these to the PO team? I'm also wondering if in these cases we shouldn't even allow the "Accept" option?
> 
> ![Screen Shot 2020-10-16 at 12 25 42 PM](https://user-images.githubusercontent.com/11698457/96283798-b9250480-0faa-11eb-9f01-64b5d631344e.png)

Yes I sent the point of contact a link to these screenshots and they didn't have any issues. Clicking accept is what triggers the red box to pop up. Are you talking about disabling the button in this case?